### PR TITLE
Create storage-fund.md

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

**Review by econmysten — APPROVED** [review]

File: doc/src/learn/tokenomics/storage-fund.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+More specifically, the storage fund has three key features: 
+
+- It is funded by past transactions and functions as a tool for shifting gas fees across different epochs. This ensures that future validators are compensated for their storage costs by the past users who created those storage requirements in the first place.
+- It pays out only the returns on its capital and does not distribute its principal. That is, in practice, it is as if validators were able to borrow the storage fund’s SUI as additional stake and keep the majority of stake rewards (a $\gamma$). But note that validators do not receive funds directly from the storage fund. This guarantees the fund never loses its capitalization and can survive indefinitely. This feature is further buttressed by the $(1-\gamma)$ of stake rewards reinvested in the fund.
```
Comment (huitseeker) [inline doc/src/learn/tokenomics/storage-fund.md:19]: Weirdly enough, I don't get the greeks formatted correctly here.

Comment (Clay-Mysten) [reply doc/src/learn/tokenomics/storage-fund.md:19]: > Weirdly enough, I don't get the greeks formatted correctly here.

@econmysten can you take a look? Thanks!

Comment (Clay-Mysten) [reply doc/src/learn/tokenomics/storage-fund.md]: > this seems to be a github problem. the math typsetting does not appear to work in bulleted lists?

I confirmed this is true by moving the lines out of the list. I tried some hacks (escaping) from this page first to no avail:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

Comment (econmysten) [reply doc/src/learn/tokenomics/storage-fund.md]: figured out a fix, leaving an empty line between the bullet points

Comment (Clay-Mysten) [reply doc/src/learn/tokenomics/storage-fund.md]: > figured out a fix, leaving an empty line between the bullet points

Nice. LGTM.

**Pull request merged**